src/common/components/status/TxPegin.vue

```diff
@@ -269,11 +269,15 @@ export default defineComponent({
     }
 
     function setMessage() {
```

Comment:
What about removing this function at all? looks like it only emits an event that's not used any more.

Reply:
this function is in use, is called by another component or when the front-end receives the back-end data.